lib/constants/signuplogin_constants.dart

```diff
@@ -0,0 +1,25 @@
+//Welcome in the build
+const String welcomeMessage = "Welcome to Aspire";
+
+//Account Verify
+const String vertifyaccountTitle = "Verify your account";
```

Comment:
Make sure to follow the camelCase format. This should be `verifyAccountTitle`.

Reply:
Got ya